.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,47 +1,35 @@
-<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
-<!-- The text between the arrows are comments - they will not be visible on your PR. -->
+<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
 
 ## About the PR
-<!-- What did you change in this PR? -->
+<!-- What did you change? -->
 
 ## Why / Balance
-<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
+<!-- Why was it changed? Link any relevant discussions or issues. -->
```

Comment:
i'd rather have the sentence "please discuss how this would affect game balance" than simply "why was it changed"

the former is a much better question for encouraging a genuine explanation.